### ISSUE TITLE
Publish to HomeBrew

### ISSUE THREAD

**Issue by Goooler** [upscayl/upscayl#270]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#faq-).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.

### Description

Publish the cask to HomeBrew to be easier to install and update.

### Alternatives

Nope.

**Comment by aaronliu0130** [comment]
Duplicate of #191 and #34. This is currently blocked by our unwillingness to pay $99 a year for code signing.
> I have checked that this issue isn't addressed in any other issue or pull request.

![gifgit](https://user-images.githubusercontent.com/11874211/218282922-efab6795-84af-4cfb-9400-c28c8729ea78.gif)

**Comment by NayamAmarshe** [comment]
I'd probably publish Upscayl on AppStore instead ;)

**Comment by aaronliu0130** [comment]
If you're talking about the mac app store, I'm pretty sure that also requires $99 a year.

**Comment by NayamAmarshe** [comment]
> If you're talking about the mac app store, I'm pretty sure that also requires $99 a year.

That's alright :)

**Comment by pavelzw** [comment]
I have upscayl on my personal homebrew tap. You can install it using `brew install --no-quarantine pavelzw/pavelzw/upscayl`

Cask code: https://github.com/pavelzw/homebrew-pavelzw/blob/main/Casks/upscayl.rb

**Comment by NayamAmarshe** [comment]
> I have upscayl on my personal homebrew tap. You can install it using `brew install --no-quarantine pavelzw/pavelzw/upscayl`
> 
> Cask code: https://github.com/pavelzw/homebrew-pavelzw/blob/main/Casks/upscayl.rb

Is it possible to make it official?

**Comment by pavelzw** [comment]
Sure, we can also move it in this organization. For this, we need a separate repository in this organization called `homebrew-<tapname>`, for example `homebrew-upscayl`. When we put the cask code there, we will be able to install it via `brew install --no-quarantine upscayl/upscayl/upscayl` or via

```bash
$ brew tap upscayl/upscayl
$ brew install --no-quarantine upscayl
```

**Comment by akrabu** [comment]
I think the SHA256 hash needs updated for the latest release...

```
Error: upscayl: SHA256 mismatch
Expected: 8cd9db46126d8108d99df0da9356748e0d1a13b877a5ed017ce2e56804e85eb1
  Actual: 632506a5d5c92593ac9f81707c067c0846d1b80647b3b3864748a9839d6dcfab
    File: /Users/akrabu/Library/Caches/Homebrew/downloads/47b6f106ffe3290c80bc1fb4534891646119ca5c2e743273dedf9d80554c3970--upscayl-2.11.0-mac.dmg
```

**Comment by aaronliu0130** [comment]
It's open source. You may be the one to change that!

**Comment by akrabu** [comment]
This is fair - and I would, if I knew how 😅

**Comment by NayamAmarshe** [comment]
Hi @pavelzw I've invited you to the repository https://github.com/upscayl/homebrew-upscayl/

Sorry for delaying it for so long 🙏

**Comment by aaronliu0130** [comment]
There's a homebrew cask in homebrew's official repos, I don't think we need to make a separate repository for this. There's a contributing guide and I'm going to start bumping the shasum now.

**Comment by NayamAmarshe** [comment]
> There's a homebrew cask in homebrew's official repos, I don't think we need to make a separate repository for this. There's a contributing guide and I'm going to start bumping the shasum now.

I think it's still good to have it under the org, just in case. Individual contributions are always welcome of course but if we can make it official, it's better for everyone.

**Comment by aaronliu0130** [comment]
Homebrew Casks are usually centralized in https://github.com/Homebrew/homebrew-cask, where I did submit 2.9.9 a couple months ago. In my opinion, duplicated work across different repositories isn't very good. I don't see how this is individual vs official.

**Comment by NayamAmarshe** [comment]
> Homebrew Casks are usually centralized in https://github.com/Homebrew/homebrew-cask, where I did submit 2.9.9 a couple months ago, which allows for installation without adding additional sources. In my opinion, duplicated work across different repositories isn't very good. I don't see how this is individual vs official.

Ah ok, sorry I misunderstood how it works then.
